charts/rancher-vsphere-cpi/questions.yaml

```diff
@@ -50,12 +50,12 @@ questions:
     group: Configuration
     show_subquestion_if: true
     subquestions:
-      - variable: labels.region
+      - variable: vCenter.labels.region
```

Comment:
A few lines up it's set to `default: true` - do we really want it to default to true? I know it's technically outside the scope of this PR, but previously we did not generate any labels so not sure why we want to default to true.

Reply:
I was just considering this. Awhile back, we had a bug where if `vSphere Tags` was set to true but there was no user input in the region/zone fields then the CPI chart would fail to deploy. I thought this scenario is kind of setup erroneously because not every user will have zoning setup on the backend infra or want their workloads deployed in a specific location. This is not the default so generating the labels section shouldn't be set to `true` as default, it should be false. This never reached the UI team I think so it was never fixed. IMO that should be updated to `default:false`

Reply:
Yeah, let's update this default to `false` to match the "pre-change" behavior. Including the default in `values.yaml`.